### PR TITLE
dts: xtensa: intel_adsp_ace15/ace20/tgph: correct SSP definition

### PR DESCRIPTION
**Pull request by kv2019i**
Fix the SSP definitions for multiple adsp platforms.

### CODE REVIEW

File: dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi [hunk 1]
```diff
@@ -293,51 +293,6 @@
...
-				   0x00079C00 0x200>;
-			interrupts = <0x04 0 0>;
-			interrupt-parent = <&ace_intc>;
-			dmas = <&lpgpdma0 12
-					&lpgpdma0 13>;
-			dma-names = "tx", "rx";
-			power-domain = <&io0_domain>;
-			status = "okay";
-		};
-
```
Comment (plbossart) [inline dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi:340]: I don't think this is correct. APL has 6 SSPs. That's the only case we have where all 6 SSPs needs to be exposed.

**Review by plbossart — CHANGES_REQUESTED** [review]

File: dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi (hunk 1 above)
Comment (plbossart) [reply dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi:340]: never mind, I read cavs15, need more coffee.

**Review by plbossart — APPROVED** [review]

**Comment by dnikodem** [comment]
@kv2019i  What about the cavs25.dtsi file? Shouldn't it also be changed?

**Review by tmleman — APPROVED** [review]

**Comment by kv2019i** [comment]
@dnikodem the cavs25/tgp definition is actually correct, it has 6 SSP instances. the tgph variant has 3.

**Review by dnikodem — APPROVED** [review]

**Review by dcpleung — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Pull request merged**